src/ui/js/Game.js

```diff
@@ -2847,6 +2769,123 @@ Game.gamePlayerDice = function(player, player_active) {
   return allDice;
 };
 
+Game.isDieClickable = function(isPlayerActive, player, die) {
+  // Find out whether this die is clickable: it is if the player
+  // is active and this particular die is not dizzy
+  if (isPlayerActive) {
+    if (die.properties.indexOf('Dizzy') >= 0) {
+      return false;
+    } else if ((player != 'player') && (die.skills.indexOf('Warrior') >= 0)) {
+      return false;
+    } else if (die.properties.indexOf('WasJustCaptured') >= 0) {
+      return false;
+    } else {
+      return true;
+    }
+  } else {
+    return false;
+  }
+};
+
+Game.createBorderDiv = function(color) {
+  return $('<div>', {
+    'class': 'die_border',
+    'style': 'border: 2px solid ' + color,
+  });
+};
+
+Game.createContainerDiv = function(
```

Comment:
Nit: since createBorderDiv and createContainerDiv are functions to create borders and containers *for dice*, IMO the function names should contain the word "Die" --- Game.js is a big module with lots of stuff going on, so this could actually reduce confusion.